app/components/webauthn_input_component.rb

```diff
@@ -36,4 +41,12 @@ def initial_hidden_tag_options
       { class: 'js' }
     end
   end
+
+  def show_desktop_ft_unlock_option?
+    if desktop_ft_unlock_option? && I18n.locale == :en
```

Comment:
Did we decide if we're doing English-only for this test?

Reply:
Yeah we are doing English only because it has not been implemented and there does not appear to be a ticket for implementing translations

Reply:
> Yeah we are doing English only because it has not been implemented and there does not appear to be a ticket for implementing translations

Can you expand on what you mean by "it" in "it has not been implemented" ?

Reply:
Sorry, "it" being the content that was provided to the team by Department of State.